### PR TITLE
Add how-to page to compile a cluster

### PR DESCRIPTION
**Pull request by ccremer**
* Adds a dedicated how-to page to compile clusters including setting environment variables.

## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Update the documentation.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/compile-catalog.adoc [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+----
+
+== Compilation
+
+.Choose cluster
+. Get the Cluster ID from Lieutenant.
++
+[source,bash]
+----
+kubectl get cluster -n <lieutenant-instance>
```
Comment (simu) [inline docs/modules/ROOT/pages/how-tos/compile-catalog.adoc]: Alternatively `commodore catalog list -v`

**Review by simu — APPROVED** [review]
LGTM, but I'm not sure this needs to be separate from "Change a parameter".

File: docs/modules/ROOT/pages/how-tos/compile-catalog.adoc (hunk 1 above)
Comment (ccremer) [reply docs/modules/ROOT/pages/how-tos/compile-catalog.adoc]: Would the LIEUTENANT_NS variable still be needed?

**Comment by ccremer** [comment]
> LGTM, but I'm not sure this needs to be separate from "Change a parameter".

They're very similar indeed. I could update the change-parameter guide so it builds on top of compile-catalog.

File: docs/modules/ROOT/pages/how-tos/compile-catalog.adoc (hunk 1 above)
Comment (ccremer) [reply docs/modules/ROOT/pages/how-tos/compile-catalog.adoc]: I just compiled without LIEUTENANT_NS and it still worked.

**Pull request merged**